packages/flutter_tools/lib/src/commands/build_fuchsia.dart

```diff
@@ -34,30 +42,32 @@ class BuildFuchsiaCommand extends BuildSubCommand {
   };
 
   @override
-  String get description => 'build the Fuchsia target (Experimental).';
+  String get description => 'Build the Fuchsia target (Experimental).';
 
   @override
   Future<FlutterCommandResult> runCommand() async {
     Cache.releaseLockEarly();
     final BuildInfo buildInfo = getBuildInfo();
     final FlutterProject flutterProject = FlutterProject.current();
     if (!platform.isLinux && !platform.isMacOS) {
-      throwToolExit('"build Fuchsia" only supported on Linux and MacOS hosts.');
+      throwToolExit('"build fuchsia" is only supported on Linux and MacOS hosts.');
     }
     if (!flutterProject.fuchsia.existsSync()) {
-      throwToolExit('No Fuchsia project configured.');
+      throwToolExit('No Fuchsia project is configured.');
     }
     final String appName = flutterProject.fuchsia.project.manifest.appName;
     final String cmxPath = fs.path.join(
         flutterProject.fuchsia.meta.path, '$appName.cmx');
     final File cmxFile = fs.file(cmxPath);
     if (!cmxFile.existsSync()) {
-      throwToolExit('Fuchsia build requires a .cmx file at $cmxPath for the app');
+      throwToolExit('The Fuchsia build requires a .cmx file at $cmxPath for the app.');
```

Comment:
Could we link to an example cmx file here?

Reply:
Added a cmx file for the stocks example.